libcontainer/container.go

```diff
@@ -56,9 +57,16 @@ type BaseState struct {
 	// InitProcessPid is the init process id in the parent namespace.
 	InitProcessPid int `json:"init_process_pid"`
 
-	// InitProcessStartTime is the init process start time.
+	// InitProcessStartTime is the init process start time in clock cycles since boot time.
 	InitProcessStartTime string `json:"init_process_start"`
 
+	// CreationTimeStamp is the unix timestamp for the start time of the container in UTC
+	CreationTimeStamp time.Time `json:"creation_timestamp"`
+
+	// CreationDirectory is the path to the bundle in which the spec for the
+	// container is loaded and also
+	CreationDirectory string `json:"creation_directory"`
```

Comment:
Comment looks like it was cut short.  Also, if we keep this I'd prefer if it were called "BundleDir" instead of "CreationDir" so it aligns with the spec. No idea what a "creation dir" is w/o reading the code/comments - while "bundle dir" would be obvious.